plasmapy/physics/transport.py

```diff
@@ -251,13 +260,21 @@ def _boilerPlate(T, particles, V):
         except Exception:
             raise ValueError("Unable to find charge of particle: "
                              f"{particles[i]}.")
+    return (masses, charges)
+
+
+def getReducedMass(masses):
     # obtaining reduced mass of 2 particle collision system
     reduced_mass = masses[0] * masses[1] / (masses[0] + masses[1])
+    return reduced_mass
```

Comment:
I'm pretty sure we could avoid having to define this on our own here by using #262 or some oneliner with [the `Particle` class](http://plasmapy.readthedocs.io/en/latest/atomic/index.html).